framework/src/auxkernels/MaterialStdVectorAux.C

```diff
@@ -34,5 +37,8 @@ MaterialStdVectorAux::~MaterialStdVectorAux()
 Real
 MaterialStdVectorAux::getRealValue()
 {
+  mooseAssert(_special_qp < _q_point.size(), "MaterialStdVectorAux.  special_qp specified as " << _special_qp << " but there are only " << _q_point.size() << " quadpoints in the element");
```

Comment:
Does it make sense to have this check be an assert? Is there ever a case where elements do not all have the same number of QPs? If so then this needs to be a runtime check that does _not_ error out, otherwise there is no way of checking the high QPs in some elements that are not provided by all elements.